iron-location.html

```diff
@@ -261,18 +261,32 @@
         return null;
       }
       var eventPath = Polymer.dom(event).path;
-      var href = null;
+      var anchor = null;
       for (var i = 0; i < eventPath.length; i++) {
         var element = eventPath[i];
         if (element.tagName === 'A' && element.href) {
-          href = element.href;
+          anchor = element;
           break;
         }
       }
+
       // If there's no link there's nothing to do.
-      if (!href) {
-        return null;
+      if (!anchor) {
+        return;
+      }
+
+      // Target blank is a new tab, don't intercept.
+      if (anchor.target === '_blank') {
+        return;
       }
+      // If the link is for an existing parent frame, don't intercept.
+      if ((anchor.target === '_top' ||
+           anchor.target === '_parent') &&
+          window.top !== window) {
+        return;
+      }
```

Comment:
Gotcha. I prefer this style because I find it easier to document and to read, admittedly at the cost of more lines.

Willing to hash this out on this at some point and abide by the consensus though.